sklearn/cluster/_kmeans.py

```diff
@@ -1286,7 +1286,7 @@ def _mini_batch_step(X, sample_weight, x_squared_norms, centers, weight_sums,
     inertia : float
         Sum of squared distances of samples to their closest cluster center.
 
-    squared_diff : numpy array, shape (n_clusters,)
+    squared_diff : numpy array of shape (n_clusters,)
```

Comment:
Use `ndarray` instead of `numpy array` (as stated).

Reply:
ok